docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
```

Comment:
This is the docker compose file that makes it easy to run the Docker image defined in the Dockerfile